02-household-queries/retrieval.py

```diff
@@ -19,9 +23,16 @@ def retrieval_call(llm, vectordb):
 
     )
 
-    print("Please state your question here: ")
-    question = input()
+    question = os.environ.get("USER_QUERY")
```

Comment:
Even if we're prompting the user later on, I don't think we should set a default for this

Reply:
I added this to facilitate testing the code with the same question, i.e., so I don't have to type or paste in the same question repeatedly between runs.

Reply:
If `USER_QUERY` is not set, then no default is set.